### DISCUSSION TITLE
Multi-line input support for custom commands?

### DISCUSSION THREAD

**Discussion by samuellawerentz** [jesseduffield/lazygit#2761] [General]
So similar to the commit description box, is there a way where we can use multi-line input for custom commands?
I have a particular commit workflow, which prompts for JIRA ticket number, type of commit, commit message and then commit description. I love how easy it is to bring our own custom workflows into lazygit using custom commands. It keeps my commits uniform and clean.

So i was wondering if i can use multi-line inputs for the commit description as used by the default commit `c` binding.

**Comment by ruudk** [comment]
I would like to do the same. It feels like the Commit Description box is a custom one. But ideally, it would be great if we could leverage that one by defining `multiline: true` or `type: textarea` or something.

**Reply by ruudk** [reply]
@stefanhaller sorry to ping you, but maybe you have ideas if this is possible, and if it is, how to do it. I would be happy to start working on a next PR. 🙂

**Reply by stefanhaller** [reply]
See here: https://github.com/jesseduffield/lazygit/pull/3548#issuecomment-2495167785
